apps/hash-api/src/graph/ensure-system-graph-is-initialized/migrate-ontology-types/migrations/007-create-api-usage-tracking.migration.ts

```diff
@@ -358,24 +340,6 @@ const migrate: MigrationFunction = async ({
       inputUnitCost: 0.00000025,
       outputUnitCost: 0.00000125,
     },
```

Comment:
I don't think this pricing info is correct @CiaranMn per https://www.anthropic.com/pricing#anthropic-api

Reply:
Haiku is 0.25 per m input tokens, 0.00000025 *1m is 0.25 so it looks right to me